models/stg_recharge__address_shipping_line.sql

```diff
@@ -25,8 +25,8 @@ final as (
         price,
         code,
         title,
-        _fivetran_deleted,
-        cast(_fivetran_synced as {{ dbt.type_timestamp() }}) as _fivetran_synced
+        _fivetran_deleted, 
```

Comment:
Yes, I have now added. Since this will also be added to new tables, I have added the `_fivetran_deleted ` field and filter to:

- ADDRESS
- ADRESS_SHIPPING_LINE
- CHARGES
- CUSTOMER
- ONE_TIME_PRODUCT
- ORDER
- PRODUCT
- SUBSCRIPTION
- SUBSCRIPTION_HISTORY